### PR TITLE
v7: Documentation | Update props used in CodePane

### PR DESCRIPTION
**Pull request by chrisipanaque**
### Description
- [x] Update docs to reflect new `highlightRanges` prop introduced in #934 and its updates in #959 

#### Type of Change

- [x] Documentation update

### Checklist

- [x] I have made corresponding changes to the documentation
- [x] My code follows the style guidelines of this project (I have run `yarn format`)
- [x] New and existing unit tests pass locally with my changes (I have run `yarn test`)
- [x] My changes generate no new warnings

### CODE REVIEW

File: docs/content/api-reference.md [hunk 1]
```diff
@@ -90,32 +90,34 @@ Appear is a component that makes a component animate on the slide on key press.
...
-| `highlightEnd`   | PropTypes.number                                                                           | `2`                   |
-| `highlightStart` | PropTypes.number                                                                           | `1`                   |
-| `indentSize`     | PropTypes.number                                                                           | `2`                   |
-| `language`       | PropTypes.string                                                                           | `javascript`          |
-| `theme`          | [Prism Theme](https://github.com/FormidableLabs/prism-react-renderer/tree/main/src/themes) | —                     |
+Additionally, the `highlightRanges` prop accepts an array that can be used to highlight certain ranges of code. This array can contain a range of two numbers `[1, 3]` which will highlight lines 1 through 3. It can also contain a list of sub-arrays which will be considered as a list of ranges `[[1, 3], [6, 8], [10, 15]]`, and if in between the list of ranges, you would like to only highlight a single line, then a number can be included within this list `[[1, 3], 5, [6, 8], [10, 15], 20]`.
```
Comment (carlos-kelly) [inline docs/content/api-reference.md:97]: I would add a note that each range is a step in the slide's animation count. They are displayed with each advance.

**Review by carlos-kelly — APPROVED** [review]
Looks great, perhaps I would add one note.

File: docs/content/api-reference.md (hunk 1 above)
Comment (ryan-roemer) [reply docs/content/api-reference.md:97]: Suggest to minorly change:

```md
, and if in between the list of ranges, you would like to only highlight a single line, then a number can be included within this list `[[1, 3], 5, [6, 8], [10, 15], 20]`.
```

with

```md
. Array values can even be mixed to include sub-arrays (for multiple lines) and numbers (for single lines), e.g., `[[1, 3], 5, [6, 8], [10, 15], 20]`.
```

**Review by ryan-roemer — APPROVED** [review]
Nice docs! Agree with @carlos-kelly comment and have one other minor wording suggestion.

**Comment by chrisipanaque** [comment]
@carlos-kelly , @ryan-roemer 

Docs updated, thank you!

Please let me know if any additional changes:
Screen Shot 2020-12-08 at 10 02 23 AM

**Pull request merged**